hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieAppendHandle.java

```diff
@@ -303,8 +404,8 @@ private Writer createLogWriter(Option<FileSlice> fileSlice, String baseCommitTim
         .onParentPath(FSUtils.getPartitionPath(hoodieTable.getMetaClient().getBasePath(), partitionPath))
         .withFileId(fileId).overBaseCommit(baseCommitTime)
         .withLogVersion(latestLogFile.map(HoodieLogFile::getLogVersion).orElse(HoodieLogFile.LOGFILE_BASE_VERSION))
+        .withFileSize(latestLogFile.map(HoodieLogFile::getFileSize).orElse(0L))
         .withSizeThreshold(config.getLogFileMaxSize()).withFs(fs)
-        .withLogWriteToken(latestLogFile.map(x -> FSUtils.getWriteTokenFromLogPath(x.getPath())).orElse(writeToken))
```

Comment:
I am seeing the log version start from 2 in my local test. I guess it's probably we removed the write token here and initializing the token will increment the log version by 1. The `HoodieLogFile.LOGFILE_BASE_VERSION` is 1.

Reply:
I will fix this. should not happen